pyproject.toml

```diff
@@ -19,6 +19,7 @@ dill = "~0.3.4"
 meinheld = "~1.0.2"
 Werkzeug = "~2.0.3"
 crypto-cpp-py = "^1.0.4"
+cloudpickle = "^2.1.0"
```

Comment:
Should we remove `dill`?

Reply:
https://github.com/Shard-Labs/starknet-devnet/pull/120/commits/0550b43b5ed3a1e71eaf2133f88d695829b04e88